auth/developer-single-sign-on-across-mobile-services.md

```diff
@@ -0,0 +1,59 @@
+# Abstract
+
+This document outlines a proposal for Single Sign On across Mobile Services & Mobile Enabled Services provisioned on OpenShift. This proposal is *not* intended as a feature for end users of Mobile Apps. It *is* intended as a development time feature for Mobile developers.
```

Comment:
I know it's very small stuff but in most of the cases I have seen name`Single sign-on` 
For example: https://en.wikipedia.org/wiki/Single_sign-on

Reply:
`This proposal is *not* intended as a feature for end users of Mobile Apps. It *is* intended as a development time feature for Mobile developers.` 

Any example will be good here to make sure that people will understand it.